src/persist-client/src/batch.rs

```diff
@@ -458,20 +443,13 @@ pub(crate) const INLINE_WRITES_TOTAL_MAX_BYTES: Config<usize> = Config::new(
 
 impl BatchBuilderConfig {
     /// Initialize a batch builder config based on a snapshot of the Persist config.
-    pub fn new(value: &PersistConfig, shard_id: ShardId) -> Self {
+    pub fn new(value: &PersistConfig, _shard_id: ShardId) -> Self {
```

Comment:
Can we just remove the argument if it's unused?

(maybe it's used in a later commit, we'll see!)

Reply:
Since we've needed it so often in the past I'm happy with keeping it!